src/js/columnsToggle.js

```diff
@@ -0,0 +1,87 @@
+(function ($) {
+  /**
+   * Function that adds column sizes to column titles based on
+   * the selected design.
+   *
+   * @param design
+   * @param leftColumn
+   * @param rightColumn
+   */
+  function toggleFields(design, leftColumn, rightColumn) {
+    // Get the selected design.
+    let selectedDesign = design.val();
+    let left = '';
+    let right = '';
+
+    // Apply the titles for the columns size element.
```

Comment:
There is some issue still here - the js applies the size indicators only to the first paragraph and the rest are left without:

<img width="1423" alt="Screenshot 2021-02-05 at 16 28 08" src="https://user-images.githubusercontent.com/2276077/107046811-b9524580-67cf-11eb-8d5e-7a2354188276.png">

Reply:
This also works now 💪